contracts/src/Nodes.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: MIT
```

Comment:
This smart contract is being thrown out and rewritten. Feel free to ignore for now